Gcpe.Hub.API/Startup.cs

```diff
@@ -137,7 +132,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             // app.UseHttpsRedirection();
 
             // temporary CORS fix

```

Comment:
Is this still temporary?

Reply:
I think we need to lock it down to only gov.bc.ca hosts but will have to investigate further. Do you have any experience with that? I think we need to have the SetIsOriginAllowed to check that the host is actually allowed. I just don't know best practices for where to put that